test/test_manifest.json

```diff
@@ -4189,6 +4189,12 @@
        "rounds": 1,
        "type": "eq"
     },
+    {  "id": "issue10122",
+       "file": "pdfs/issue10122.pdf",
+       "md5": "80a0a0e34433c5b3fd4a57778bb2b4b4",
+       "rounds": 1,
+       "type": "eq"
```

Comment:
You also need https://github.com/mozilla/pdf.js/blob/3404e5ee1919fda1ce7ce5908db19f7e82f63177/test/test_manifest.json#L4450 here, otherwise the annotation is not rendered in the reference tests.